src/main/java/io/appform/jsonrules/expressions/JsonPathBasedExpression.java

```diff
@@ -39,21 +39,26 @@
     private String path;
     private PreOperation<?> preoperation;
     private static final ObjectMapper mapper = new ObjectMapper();
+    private Boolean defaultResult;
```

Comment:
As discussed, you can override the default value in the super class constructor call.